bedrock/firefox/templates/firefox/features/private-browsing.html

```diff
@@ -42,6 +51,26 @@ <h2>{{ _('Browse without a trace') }}</h2>
       </div>
     </section>
 
+    {% if (variation == 'a') and (request.locale in ['en-US', 'en-GB', 'en-CA', 'en-AU']) %}
+    <section class="key-feature private-browsing-feature-new">
+      <div class="key-feature-content">
+        <h2>Open a new, blank Private Window</h2>
+        <p>
+          Click the menu button <img src="{{ static('img/firefox/election/icon-menu.svg') }}" alt="Open menu" height="12" width="14"> and then click New Private Window.
```

Comment:
Also not a blocker, just an idea - we could go one step further here in the future perhaps, by using UITour to directly open the menu and highlighting the Private Browsing icon, as opposed to just describing what to do. We used to do this on the old `/firefox/private-browsing/` page as the main CTA.